docs/resources/test.md

```diff
@@ -115,10 +115,12 @@ resource "kentik-synthetics_test" "minimal-hostname-test" {
     ]
     ping {
       period = 60
+      count = 1
     }
     trace {
       period   = 60
       protocol = "udp"
```

Comment:
With `protocol = "udp"`, the `trace` config must define also `port`. I suggest to change the example and use `protocol = "icmp"`. Applies to all instances.